crates/biome_js_analyze/src/assist/source/organize_imports.rs

```diff
@@ -1,143 +1,328 @@
-    /// They will be sorted like this:
-    ///
-    /// ```ts
-    /// import { expect } from "bun:test";
-    /// import assert from "node:assert";
-    /// import { mock, test } from "node:test";
-    /// import express from "npm:express";
-    /// import { sortBy } from "virtual:utils";
-    /// import { VERSION } from "https://deno.land/std/version.ts";
-    /// import React from "react";
-    /// import { secret } from "/absolute/path";
-    /// import { internal } from "#internal";
-    /// import aunt from "../aunt";
-    /// import uncle from "../uncle";
-    /// import sibling from "./sibling";
-    /// import imageUrl from "url:./image.png";
+    /// Chunks also ends as soon as a statement or a side-effect import (also called bare import) is encountered.
+    /// The following example contains three chunks.
+    ///
+    /// ```js,ignore
+    /// import A from "a";
+    /// import "x";
+    /// import * as B from "b";
+    /// function f() {}
+    /// import * as C from "c";
     /// ```
     ///
-    /// You can apply the sorting in two ways: via [CLI](#import-sorting-via-cli) or [VSCode extension](#import-sorting-via-vscode-extension).
+    /// Finally, a last way of intentionally start a new chunk is to use a detached comment.
+    /// A **detached comment** is a comment separated of an import or export with a blank line.
+    /// Comments that are not separated of an import or export by a blank line are **attached comments**.
+    /// The following example contains a detached comment that split the imports into two chunks.
     ///
-    /// ## Grouped imports
+    /// ```js,ignore
+    /// // Attached comment 1
+    /// import A from "a";
+    /// // Attached comment 2
+    /// import * as B from "b";
     ///
-    /// It's widespread to have import statements in a certain order, primarily when you work on a frontend project, and you import CSS files:
+    /// // Detached comment
     ///
-    /// ```js
-    /// import "../styles/reset.css";
-    /// import "../styles/layout.css";
-    /// import { Grid } from "../components/Grid.jsx";
+    /// import * as C from "c";
     /// ```
     ///
-    /// Another common case is import polyfills or shim files, that needs to stay at the top file:
+    /// The import and export organizer ensures that chunks are separated from each other with a blank lines.
+    /// Only side-effect imports adjacent to a chunk are not separated by a blank line.
+    ///
+    /// Imports or exports of a chunk are sorted together.
+    /// Import and export sorting uses several layer of order.
+    ///
+    /// ## Import and export groups
+    ///
+    /// Imports or exports of a chunk are divided into groups.
+    /// By default, it exists a single group: the implicit group.
+    ///
+    /// You can define extra groups using the `groups` option.
+    /// The organizer provides predefined groups and allows you to create new groups based on a glob or a list of globs.
+    /// The predefined groups are:
```

Comment:
A couple of pointers:
- before, you used the word "implicit"; now, the word "predefined" is used. Let's stick with one. 
- Before, you mentioned **one** group, but now it seems there are two.

Reply:
The implicit/default group is not the same concept. Predefined groups are matcher like globs. The default implicit/default group is just an all-catching group.

Reply:
Your reply made me more confused 😕 not sure if I can provide a good suggestion. See if you can rephrase it

Reply:
Ok I will try to improve. I can also add more predefined groups in order to build the default order using the `groups` option.